### PR TITLE
Add worker and statuser profiles

### PR DESCRIPTION
**Pull request by adiabramovitch**
I think that there are linter changes in the compose file, sorry :)

### CODE REVIEW

File: Dockerfile.worker.dev [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    CGO_ENABLED=0
+
+WORKDIR /backend
+RUN mkdir "/build"
+
+COPY . .
+RUN go install github.com/githubnemo/CompileDaemon@latest
+RUN make prep GO=go
+
+ENTRYPOINT ["bash", "-c", "go run ./cmd/pbackend worker && $HOME/go/bin/CompileDaemon -build='go build -buildvcs=false -o /build/pbackend ./cmd/pbackend' -command='/build/pbackend worker'"]
```
Comment (amirfefer) [inline Dockerfile.worker.dev]: you can use just the `CompileDaemon `
```suggestion
ENTRYPOINT ["bash", "-c",  "$HOME/go/bin/CompileDaemon -build='go build -buildvcs=false -o /build/pbackend ./cmd/pbackend' -command='/build/pbackend workeri'"]
```

**Comment by lzap** [comment]
Try this if it speeds things up: https://github.com/RHEnVision/provisioning-compose/pull/14

**Comment by lzap** [comment]
How about this: https://github.com/RHEnVision/provisioning-backend/pull/658

File: compose.yml [hunk 2]
```diff
@@ -143,6 +146,34 @@ services:
     volumes:
       - ./backend/:/backend
 
+  statuser-dev:
+    env_file:
+      - backend.env
+    depends_on:
```
Comment (adiabramovitch) [inline compose.yml:153]: I think it is better not to include worker and statuser as part of the backend-dev profile. 
If someone wants to use worker or statuser they can just add it to the Compose command.

Comment (adiabramovitch) [reply compose.yml:153]: I see an issue, that the statuser and worker does not reload on changes even though the compile Daemon is used. can you check that please? @amirfefer

**Review by lzap — APPROVED** [review]
Not a huge fan of this weird compiling daemon, I will not use it :-)

File: Dockerfile.statuser.dev [hunk 3]
```diff
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.19
+USER 0
+ENV PROJECT_DIR=/backend \
+    CGO_ENABLED=0
+
+WORKDIR /backend
```
Comment (amirfefer) [inline Dockerfile.statuser.dev]: The volume maps to `statuser` so the WORKDIR should be `/statuser`

File: Dockerfile.worker.dev [hunk 4]
```diff
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.19
+USER 0
+ENV PROJECT_DIR=/backend \
+    CGO_ENABLED=0
+
+WORKDIR /backend
```
Comment (amirfefer) [inline Dockerfile.worker.dev]: The container's volume maps to `/worker` so the WORKDIR should be `/worker`

File: compose.yml (hunk 2 above)
Comment (amirfefer) [reply compose.yml:153]: I think it is related to the WORKDIR and the volume mapping.

Comment (adiabramovitch) [reply compose.yml:153]: Yes, it is fixed now :)

**Review by amirfefer — APPROVED** [review]
Thanks @adiabramovitch 👍

**Pull request merged**